src/OrchardCore.Modules/OrchardCore.Resources/wwwroot/Scripts/monaco/ocmonaco.js

```diff
@@ -1 +1,10 @@
-require.config({ paths: { 'vs': document.currentScript.dataset.tenantPrefix + '/OrchardCore.Resources/Scripts/monaco/vs' } });
+/*
+** NOTE: This file is generated by Gulp and should not be edited directly!
+** Any changes made directly to this file will be overwritten next time its asset group is processed by Gulp.
+*/
```

Comment:
Comments like these are not supposed to be added. We removed them from the gulpfile.js.

Reply:
I did not modify the gulpfile. Pretty sure this header is still there: 
https://github.com/OrchardCMS/OrchardCore/blob/bb3d7bb5f5b0fd6d3a959f02074afc5ff34ce2a7/gulpfile.js#L285

Reply:
Ah, I see. Thanks for the link @jtkech

Reply:
Ok @jtkech is right. The script files are not concerned so we can leave it as it is.